_docs-v4/event-display/eventTimeFormat.md

```diff
@@ -51,7 +51,22 @@ var calendar = new Calendar(calendarEl, {
   eventTimeFormat: { // like '14:30:00'
     hour: '2-digit',
     minute: '2-digit',
-    second: '2-digit'
+    second: '2-digit',
+    meridiem: false
+  }
+});
+```
+
+The `eventTimeFormat` object also accepts an hour12 property, a boolean like meridiem:
+
+```js
+var calendar = new Calendar(calendarEl, {
+// other options ...
```

Comment:
slight indentation problem here. i think linting will catch this

Reply:
updated